pages/common/git-archive.md

```diff
@@ -0,0 +1,24 @@
+# git archive
+
+> Create an archive of files from a named tree.
+> More information: <https://git-scm.com/docs/git-archive>.
+
+- Create a tar archive from the contents of the latest commit on the current branch:
+
+`git archive --verbose -o {{target.tar}} HEAD`
```

Comment:
We prefer the long argumento format:
```suggestion
`git archive --verbose --output={{path/to/file.tar}} HEAD`
```

Reply:
Will use the long argument :)
As for the file name, I used the same convention used for the `tar` command, e.g., `tar cf {{target.tar}} {{file1}} {{file2}} {{file3}}`

Reply:
Hey! Yeah, filenames can be an awkward one. Generally, the rule of thumb is that we prefer including the `path/to/` prefix. However, if you've got a number of files in a row (e.g. see the page for the `diff` command), then we drop the `path/to/` prefix to avoid the example from being unnecessarily verbose. In this case though I think we can include it.